### DISCUSSION TITLE
Another llm model

### DISCUSSION THREAD

**Discussion by RuddiRodriguez** [eyurtsev/kor#201] [Q&A] [answered]
Hi , I would like to know if is possible to use another llm , beside OpenAi, like GPT4all for example. I think saw it in the documentation , but I can not find it anymore, thanks

**Comment by eyurtsev** [comment, marked as answer]
Hi @RuddiRodriguez, Kor is integrated with LangChain, which means that you can use any LLM supported by LangChain including both chat and non-chat LLMs. See documentation here: 


https://python.langchain.com/docs/integrations/llms/
https://python.langchain.com/docs/modules/model_io/models/llms/

**Reply by RuddiRodriguez** [reply]
Hi @eyurtsev, thank you very much for your reply, it worked, thanks. However, there are a couple of issues 
1- the path of the model must be hard code, otherwise there is an error regarding load the model I think is a langchain pydantic problem related since I saw the same in others forums.
2- Not an issue but observation OpenAi models are much better with complex texts , I do not know if i need to change some parameters in GPT4all model , but so far the performance is poor. 
Thanks again for the tool and your time
